CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ Notable updates:
 
 ### Added
 * [database] added ParityDb statistics to the stats endpoint. [#2444](https://github.com/ChainSafe/forest/pull/2444)
-
+* [forest-daemon] support jemalloc as optional global allocator [#2477](https://github.com/ChainSafe/forest/pull/2477)
+* [forest-daemon] support mimalloc as optional global allocator [#2506](https://github.com/ChainSafe/forest/pull/2506)
```

Comment:
Hm, after a bit of thought, I am not sure this should be included in the changelog. It's not really a user-facing feature. It's completely internal to Forest devs - eventually we will most likely pick only one global allocator. Thoughts?

Reply:
Makes sense, let me remove them